api/net/Areas/Subscriber/Controllers/FilterController.cs

```diff
@@ -0,0 +1,149 @@
+using System.Net;
+using System.Net.Mime;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.Filter;
+using TNO.API.Models;
+using TNO.Core.Exceptions;
+using TNO.Core.Extensions;
+using TNO.DAL.Services;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Subscriber.Controllers;
+
+/// <summary>
+/// FilterController class, provides filter endpoints for the api.
+/// </summary>
+[ClientRoleAuthorize(ClientRole.Subscriber)]
+[ApiController]
+[Area("subscriber")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/filters")]
+[Route("api/[area]/filters")]
+[Route("v{version:apiVersion}/[area]/filters")]
+[Route("[area]/filters")]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Unauthorized)]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Forbidden)]
+public class FilterController : ControllerBase
```

Comment:
@jdtoombs Filters are Saved Searches.  We need to change how you implemented `My Searches` so that it uses `Filters` instead.